app/requests/remote/user.ts

```diff
@@ -105,27 +103,39 @@ export const login = async ({ldapOnly = false, loginId, mfaToken, password}: Log
 };
 
 export const loadMe = async ({deviceToken, user}: LoadMeArgs) => {
-    let currentUser: RawUser = user;
+    let currentUser = user ?? undefined;
```

Comment:
```suggestion
    let currentUser = user;
```

why would you force it to be undefined?

Reply:
Corrected.